### ISSUE TITLE
about how the reference view and target view work

### ISSUE THREAD

**Issue by djx99** [kxhit/EscherNet#27]
Thank you for your excellent work, and I have a few questions about how the reference view and target view work in your model:

- Which stage of the stable diffusion model the reference view and target view images correspond to?

- Specifically, which images were used for the noise addition and denoising process?

- What loss function did you employ during the training process?

-  Additionally, I have attached a diagram of the stable diffusion process for reference. Am I correct in understanding that the target image is noised, while the reference image is used as a conditioning input?

I apologize in advance if my questions come across as naive, as I am relatively new to this field. Thanks for your time.
![Image](https://github.com/user-attachments/assets/4efca003-caa8-40f4-98ce-3fa3b5205e32)

**Comment by kxhit** [comment]
Hi all the details are in the open-sourced code and the paper. Let me know if there is still issues. Thanks!
